smallrye-reactive-messaging-mqtt/pom.xml

```diff
@@ -30,12 +30,12 @@
       <artifactId>testcontainers</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
+<!--    <dependency>
```

Comment:
why is it commented?

Reply:
Sorry, removed it. Replaced with JBoss Logging